marklogic/tests/test_marklogic.py

```diff
@@ -0,0 +1,244 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import Any, List
+
+import mock
+import pytest
+from packaging import version
+
+from datadog_checks.base.stubs.aggregator import AggregatorStub
+from datadog_checks.marklogic import MarklogicCheck
+
+from .common import (
+    COMMON_TAGS,
+    INSTANCE,
+    INSTANCE_FILTERS,
+    INSTANCE_SIMPLE_USER,
+    MARKLOGIC_VERSION,
+    SERVICE_CHECKS_HEALTH_TAG,
+)
+from .metrics import (
+    FOREST_STATUS_TREE_CACHE_METRICS,
+    GLOBAL_METRICS,
+    RESOURCE_STORAGE_FOREST_METRICS,
+    STORAGE_FOREST_METRICS,
+    STORAGE_HOST_METRICS,
+)
+
+
+def _assert_metrics(aggregator, tags):
+    # type: (AggregatorStub, List[str]) -> None
+    for metric in GLOBAL_METRICS:
+        aggregator.assert_metric(metric, tags=tags)
+
+    # May take some times to be available
```

Comment:
Does that mean the test might be flaky ?

Reply:
It may happen that `marklogic.forests.compressed-tree-cache-ratio` is missing sometimes, but most of the time it's here.